Formula/pdnsd.rb

```diff
@@ -24,6 +24,43 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    Before starting "pdnsd" using "brew services" or manually, make sure that,
+    #{etc}/pdnsd.conf, #{var}/log/pdnsd.log and directory #{var}/cache/pdnsd
```

Comment:
`chown` `nobody` would not break an upgrade in this case. FWIW, I can still edit the configuration file with my account without the need for `sudo`, after which I have to `chown` to `nobody` again. So, a `brew upgrade` can overwrite the configuration file; `log` and `cache` need not be dealt with during any of `brew` operations.

I did not find another way to deal with `pdnsd` which needs access to a privileged port(53). Also, running `pdnsd` with the current login user instead of user `nobody` seemed risky. If this seems like trouble for maintenance, I would rather close this PR.